cmd/alertmanager/main.go

```diff
@@ -209,7 +209,8 @@ func run() int {
 		peerReconnectTimeout = kingpin.Flag("cluster.reconnect-timeout", "Length of time to attempt to reconnect to a lost peer.").Default(cluster.DefaultReconnectTimeout.String()).Duration()
 	)
 
-	promlogflag.AddFlags(kingpin.CommandLine, &promlogConfig)
+	a := kingpin.CommandLine.UsageWriter(os.Stdout)
+	promlogflag.AddFlags(a, &promlogConfig)
```

Comment:
```suggestion
	promlogflag.AddFlags(kingpin.CommandLine, &promlogConfig)
	kingpin.CommandLine.UsageWriter(os.Stdout)
```

Reply:
Just a friendly bump to tell that I've commited your suggestion :)